manifests/ssl_bump.pp

```diff
@@ -26,16 +26,14 @@
     'server-first',
     'splice',
     'stare',
-    'terminate']
-          $action = 'bump',
+  'terminate']
```

Comment:
This looks off. Perhaps the closing `]` should be on a new line together with `$action`.

Reply:
fixed it